CMakeLists.txt

```
@@ -1232,7 +1232,9 @@ if(USE_OPENSSL_QUIC)
   set(USE_NGHTTP3 ON)
   include_directories(SYSTEM ${NGHTTP3_INCLUDE_DIRS})
   list(APPEND CURL_LIBS ${NGHTTP3_LIBRARIES})
+  list(APPEND CURL_LIBDIRS ${NGHTTP3_LIBRARY_DIRS})
   list(APPEND LIBCURL_PC_REQUIRES_PRIVATE ${NGHTTP3_PC_REQUIRES})
+  link_directories(${NGHTTP3_LIBRARY_DIRS})
```

```suggestion
  list(APPEND CURL_LIBS ${NGHTTP3_LIBRARIES})
  list(APPEND CURL_LIBDIRS ${NGHTTP3_LIBRARY_DIRS})
  list(APPEND LIBCURL_PC_REQUIRES_PRIVATE ${NGHTTP3_PC_REQUIRES})
  include_directories(SYSTEM ${NGHTTP3_INCLUDE_DIRS})
  link_directories(${NGHTTP3_LIBRARY_DIRS})
  if(NGHTTP3_CFLAGS)
    string(APPEND CMAKE_C_FLAGS " ${NGHTTP3_CFLAGS}")
  endif()
```

Nice catch! What about we also respect `CFLAGS` and move `include_directories()` (formatting nit) to have this block synced exactly with the other similar block within the file (at arund line 1165)?